config/flexibility_order.yml

```diff
@@ -21,5 +21,7 @@
 - pumped_storage
 - power_to_gas
 - power_to_heat_industry
+- power_to_heat_district_heating_boiler
+- power_to_heat_district_heating_heatpump
```

Comment:
Is it definitely the case that a user will want to prioritise one of these over the other, rather than just having a "Power to heat for district heating" option? For industry we just have the one option for four technologies, and excess electricity is divided fairly between them based on the capacity of each.

Reply:
Yes that's the case!